.env

```diff
@@ -77,6 +77,9 @@ ESPRESSO_DEMO_SEQUENCER_LIBP2P_PORT_2=7002
 ESPRESSO_DEMO_SEQUENCER_LIBP2P_PORT_3=7003
 ESPRESSO_DEMO_SEQUENCER_LIBP2P_PORT_4=7004
 
+# The demo uses the mock stake table contract, only capacity 10 is supported.
+ESPRESSO_SEQUENCER_STAKE_TABLE_CAPACITY=10
```

Comment:
can we add this to public-env-vars-toml as well

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1519/commits/9010557987d83bfbfc5fc1b16a5123a003a8d643